app/src/main/res/layout/activity_main.xml

```diff
@@ -28,6 +28,6 @@
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:navGraph="@navigation/mobile_navigation" />
+        app:navGraph="@navigation/transfer_navigation" />
```

Comment:
why you change the navigation file name here. it should be inside the transfer module not here.